internal/metrics/workload.go

```diff
@@ -78,7 +83,11 @@ func (wrkM *WorkloadMetrics) WorkloadStarted(workloadName string, report []*podm
 			interval = cfg.Metrics.Interval
 		}
 		// log for this is part of the AddTarget function
-		wrkM.daemon.AddTarget(workload.Name, urls, time.Duration(interval)*time.Second)
+		wrkM.daemon.AddFilteredTarget(
```

Comment:
Can we remove `AddTarget` at this point?

Reply:
Removed the AddFilteredTarget and rename to AddTarget.